example_settings_icarus_gateway_sbnd.conf

```diff
@@ -0,0 +1,66 @@
+CHANNEL_MAP_FNAME="/web/sites/s/sbn-online.fnal.gov/cgi-bin/minargon/minargon/channel_map.txt"
```

Comment:
Can you remove this from the pull request? It has some internal fermilab information that we should post publicly.